app/routes/flush.py

```diff
@@ -56,45 +60,49 @@ def flush_data():
     return Response(status=403)
 
 
-def _submit_data(user):
-    answer_store = get_answer_store(user)
+def _submit_data(user: User) -> bool:
+    questionnaire_store = get_questionnaire_store(user.user_id, user.user_ik)
 
-    if answer_store:
-        questionnaire_store = get_questionnaire_store(user.user_id, user.user_ik)
-        answer_store = questionnaire_store.answer_store
+    # Type ignore: The presence of an answer_store implicitly verifies that there must be metadata populated and thus can safely be used non-optionally.
+    # Where 'type: ignore' has been used for metadata, it is because the invoked function expects a non-optional MetadataProxy.
+    if questionnaire_store and questionnaire_store.answer_store:
         metadata = questionnaire_store.metadata
-        response_metadata = questionnaire_store.response_metadata
-        progress_store = questionnaire_store.progress_store
-        list_store = questionnaire_store.list_store
         submitted_at = datetime.now(timezone.utc)
         schema = load_schema_from_metadata(
-            metadata=metadata, language_code=metadata.language_code
+            metadata=metadata, language_code=metadata.language_code  # type: ignore
         )
 
         router = Router(
```

Comment:
Minor, but would be good to use key word args for things like Router as we do in other places

Reply:
Sure thing, will do this for Router and other similarly invocations/instantiations with many params, that have been touched n this PR.

I've seen some methods have used the kwarg enforcement in their decorations `mymethod(*, x: int, y: str)` , would it be worth adding this to the areas I add kwarg usage like Router?

Reply:
Have added kwarg usage of Router and there was only really one other method which wasn't using kwargs and could benefit from it, so added there. 
Didn't enforce kwargs on Router as the tests were failing because they didn't use kwargs and that didn't feel like a rabbit hole worth going down